roles/ojs/templates/script.exp.j2

```diff
@@ -43,7 +43,7 @@ if {$force_conservative} {
 
 
 set timeout -1
-spawn php7.4 tools/install.php
+spawn php8.1 tools/install.php
```

Comment:
@sandbergja was wondering if ojs should be protected at php7?

Reply:
I got the blessing of RDSS to move on with this